upgrade.sh

```diff
@@ -99,6 +99,9 @@ fi
 # cray-conman needs to be removed if it exists.
 undeploy -n services cray-conman
 
+# In CSM 1.6.0 the trs operator is obsolete and thus is removed here.
+undeploy -n operators cray-hms-trs-operator
+
```

Comment:
Need these lines deleted if the operator will have already been undeployed in a previous version.

Reply:
what do you do if you dont know?

Reply:
PR to remove the undeploy step in the main branch: https://github.com/Cray-HPE/csm/pull/1868/files

Reply:
FYI If that line is left in the upgrade script doesn't look like it will fail, as its able to handle missing charts (like if the script was re-ran): https://github.com/Cray-HPE/csm/blob/main/upgrade.sh#L30-L38

Reply:
Need 1.6 changed to 1.5.

Reply:
This PR to main will remove the line: https://github.com/Cray-HPE/csm/pull/1868/files